redbadger/continous-compliance
# Copy test results into a compliance folder

Closes #16 

## How to test

- Go to `count-dracula` and fetch remote branches `git fetch` 
- Checkout to the continuous compliance test branch `git checkout test-continous-compliance`
- Create an empty commit `git commit --allow-empty -m "Trigger continous-compliance CI/CD"`
- Check action running [here](https://github.com/redbadger/count-dracula/actions)

![](https://media.giphy.com/media/UhGjX7VsHOgi4/giphy.gif)
